### ISSUE TITLE
NuGet package naming

### ISSUE THREAD

**Issue by akamyshanov** [connamara/quickfixn#627]
First of all, congratulations and thank you for finally publishing the project on NuGet 🎉 

I fully understand that this is very subjective, however here are my 2 cents.
I think that the package IDs should have a different naming structure. Typically, dots in .NET world represent namespace or subproject separators. This means that [QuickFIXn.FIX4.1](https://www.nuget.org/packages/QuickFIXn.FIX4.1/) is logically divided into:
* QuickFIXn (Project name, OK)
* FIX4 (???)
* 1 (???)

This is even worse with [QuickFIXn.FIX5.0SP2](https://www.nuget.org/packages/QuickFIXn.FIX5.0SP2/):
* QuickFIXn
* FIX5
* 0SP2

I think that the project and assembly naming structure is much more clear, e.g:
* `QuickFIXn.FIX44` instead of `QuickFIXn.FIX4.1`
* `QuickFIXn.FIX50SP2` instead of `QuickFIXn.FIX4.1`

So that FIX44 or FIX50SP2 represent the spec version as a single entity.

I understand that this would break backwards compatibility for those who already downloaded the packages but I strongly believe that it would be better to change the naming in the long run.
PS. NuGet allows [delisting](https://devblogs.microsoft.com/nuget/deprecating-packages-on-nuget-org/) packages.

**Comment by amandengue** [comment]
👍 
I totally agree with this

**Comment by gbirchmeier** [comment]
I agree with this too.  We'll probably announce this upcoming change in the 1.13 release and put in the READMEs, but not actually execute the change until 1.14.

**Comment by gbirchmeier** [comment]
hey @akamyshanov and @amandengue - it took a while, but see PR #944, let me know if you have any notes
